templates/web/base/admin/body.html

```diff
@@ -142,4 +142,6 @@
 </div>
 [% END %][%# Only show all the above if no errors with category form %]
 
+[% INCLUDE 'admin/_translations.html' action=c.uri_for('body', body_id) %]
```

Comment:
Now they both submit to the same place, I think this is definitely worth doing – combine them as one form (so they appear on the new body form too), update the translations if posted is `new` or `update`, and then you can have the translations appear alongside the column they're translating.